### DISCUSSION TITLE
Automazione Sensore contatto Switchbot ed accensione luci

### DISCUSSION THREAD

**Discussion by Foofighterclub** [indomus/forum#3334] [GENERALE - Discussioni generiche]
Buongiorno.
Tramite sensore di contatto, apertura/chiusura porte Switchbot, volevo utilizzare il suo sensore di movimento per accendere una luce collegata ad un relè Shelly.
Ho provato a creare una routine con Alexa, ma in questo caso il sensore è utilizzabile solamente tramite il sensore di contatto magnetico, e non con il sensore di movimento o luminosità.
Tramite l'app Switchbot, purtroppo non è possibile configurare l'accensione dello Shelly.
Con Google Home non ho trovato modo per impostare questa automazione.
Qualche consiglio?
Grazie.

**Comment by KC-inDomus** [comment]
Usa [Home Assistant](https://indomus.it/home-assistant/).
